dynosaur/tests/pass/handle-generics-on-trait-def.rs

```diff
@@ -1,10 +0,0 @@
-use dynosaur::dynosaur;
-
-#[dynosaur(DynMyTrait)]
-trait MyTrait<T> {
-    const N: i32;
```

Comment:
Maybe just remove this const, I think the rest of the test should work?

Reply:
Then the rest is identical to other that exist.